cluster/olm/ceph/generate-rook-csv.sh

```diff
@@ -119,6 +121,11 @@ function generate_csv(){
     mv "$DEFAULT_CSV_FILE_NAME" "$DESIRED_CSV_FILE_NAME"
     "${YQ_CMD_MERGE_OVERWRITE[@]}" "$DESIRED_CSV_FILE_NAME" "$ASSEMBLE_FILE_COMMON"
     "${YQ_CMD_WRITE[@]}" "$DESIRED_CSV_FILE_NAME" metadata.annotations.externalClusterScript "$(base64 <$CEPH_EXTERNAL_SCRIPT_FILE)"
+    if [ -f "$CEPH_EXTERNAL_SCRIPT_FILE" ];then
+        pushd $(dirname "$CEPH_EXTERNAL_SCRIPT_FILE") 1>/dev/null
+        python -m unittest --verbose "$(basename "$CEPH_EXTERNAL_SCRIPT_FILE" |sed -n 's@\.py$@@gp')"
```

Comment:
We need to install some python package or just this though a container...

Something like this:

```
docker run --rm -it  --entrypoint=python3 -v "/home/leseb/go/src/github.com/rook/rook/cluster/examples/kubernetes/ceph/create-external-cluster-resources.py:/create-external-cluster-resources.py:z" python -m unittest --verbose /create-external-cluster-resources.py
```

Reply:
We don't have to install anything, it uses the general `unittest` module, which comes with python (3 or 2) package...
The only criteria is to run on the same directory as the python file...

Reply:
The CI fails with: `/usr/bin/python: No module named unittest` so I assume there is something missing on that machine.

Reply:
Just an update,
I tried to install `pytest` (on my local machine) using pip3
`pip3 install pytest`
and ran the tests using following command
```
$ python -m pytest --verbose create-external-cluster-resources.py

============================================================================ test session starts =============================================================================
platform linux -- Python 3.7.7, pytest-5.4.2, py-1.8.1, pluggy-0.13.1 -- /home/testUser1/aruns/programming/python/my_env/bin/python
cachedir: .pytest_cache
rootdir: /home/testUser1/aruns/programming/python/my_env
collected 7 items                                                                                                                                                            

create-external-cluster-resources.py::TestRadosJSON::test_method_create_cephCSIKeyring_cephFSProvisioner PASSED                                                        [ 14%]
create-external-cluster-resources.py::TestRadosJSON::test_method_main_output PASSED                                                                                    [ 28%]
create-external-cluster-resources.py::TestRadosJSON::test_multi_data_pool_scenario PASSED                                                                              [ 42%]
create-external-cluster-resources.py::TestRadosJSON::test_multi_filesystem_scenario PASSED                                                                             [ 57%]
create-external-cluster-resources.py::TestRadosJSON::test_non_zero_return_and_error PASSED                                                                             [ 71%]
create-external-cluster-resources.py::TestRadosJSON::test_rbd_data_pool_name_arg PASSED                                                                                [ 85%]
create-external-cluster-resources.py::TestRadosJSON::test_valid_rgw_endpoint PASSED                                                                                    [100%]

============================================================================= 7 passed in 0.02s ==============================================================================
```

PS: using `pytest` you can run the test from any directory,
`python3 -m pytest --verbose my/test/env/newdir/create-external-cluster-resources.py`